module/applications/actor/base-sheet.mjs

```diff
@@ -661,10 +661,13 @@ export default class ActorSheet5e extends ActorSheetMixin(ActorSheet) {
       // Active Effect management
       html.find(".effect-control").click(ev => ActiveEffect5e.onManageActiveEffect(ev, this.actor));
       this._disableOverriddenFields(html);
+
+      // Item Context Menu
+      new ContextMenu(html, ".item-list .item", [], {onOpen: this._onItemContext.bind(this)});
```

Comment:
This change makes it more awkward for downstream sheet subclasses to add context menu items for observers only, for example. We should consider instead checking for `isEditable` when we are adding items to the context menu (or using the `condition` property on context menu items), in order to leave open the possibility of adding menu items in other contexts.